### ISSUE TITLE
What Is The Science Outpost Chemdispenser Really Used For: A Fun Discussion

### ISSUE THREAD

**Issue by EdXbtkOEwGw** [d3athrow/vgstation13#16571]
The Chemdispenser that was added to Science Outpost was, at first, intended for use by Xenoarchaeology, who uses it for their autistic needs. Later, Xenobotany was added to the outpost, with the intention of having them to mooch off of it.
I searched through the chemistry logs to find out exactly what use do people really give it.
Here's the Top Five Scientist-Approved Chemicals:

# 1- Chloral Hydrate
Yep. The chemical that gets made the most often in the Science Outpost is chloral hydrate. Out of 5928 rounds scanned, 3575 of them had someone make it using the roundstart outpost chemdispenser. That is **60.3%** of all rounds. This includes lowpop and short rounds, in a 2hr midpop+ round, that number may be as high as 75%.

# 2- Hyperzine
It's no secret that scientists are hyperzine junkies. Hyperzine is made in 50.8% of all rounds, and an average of 122u are made in each round that it gets made. Xenoarchaeology somewhat needs their hyperzine, but there's a Xenoarchaeologist in a lot less than 51% of all rounds... where does the rest come from?

# 3- Unstable Mutagen
It seems that the chemdispenser is used more for xenobotany than xenoarchaeology. The interesting part is seeing how much of it gets made. Mutagen is made in 33% of rounds, and on average, 105.1u are made each round that it gets made, with instances of people making 270u or more in one round. So, theoretically, a bucket full of mutagen should be enough for most players, with the possibility of getting more from elsewhere.

# 4- Anti-Toxin
Yikes. People figured out anti-toxin makes you immune to chloral, and so 148491u of anti-toxin have been made in total. It's only made 30.7% of the time, but when it gets made, a few recurrent ckeys literally empty the chemdispenser energy pumping their veins with enough antitoxin to last them for the rest of the round. The powergame is literally out of this world, lads.

# 5- Lithiumsodiumtungstate
Oh hey! It's the literal reason for the chemdispenser's existence. It's only seen 25.9% of rounds, however, and when it gets made, people only make one 30u batch in average. Hmmm. Looks like the chemdispenser could just have been replaced with a beaker of oxygen, a small vial of sodium, a syringe of lithium, and a syringe of tungsten. And... just that would have been enough for xenoarch.
But even weirder than that is that it seems half of that lithiumsodiumtungstate is never actually used. Density separated samples (the thing you use this chemical for) are only created 12% of rounds. Really makes me think...

So, now that the dust has settled, and now that you are armed with the full picture, do you think the Science Outpost Chemdispenser was a good addition?

**Comment by eplgr** [comment]
Do you think "yes, antags usually use it, we should remove it" is a valid statement?

There's no logical conclusion here, from you. You posted a lot of data that is basically useless and/or misinterpreted.
Should we also remove the whole tcomms room since 99% of the time anyone goes there is to fuck up everything? 

Whether the outpost chem dispenser was a bad idea or not, it has *nothing* to do with what you stated.

**Comment by Pathid** [comment]
Maybe instead chloral, hyperzine and mega antitox doses should be less reliable/more dangerous to use in the first place.

**Comment by EdXbtkOEwGw** [comment]
@eplgr I merely provided facts and asked for your opinion. What you bring to the table is a theory that the dispenser must be used the most often by antags, but I don't see any data or argumentation to make it a logical theory. Or a valid statement, if you will.

I figure usage stats have at least a tiny bit of relevance when discussing the thing itself, or am I wrong?

**Comment by Durgen** [comment]
The obvious solution is to add a subtype of dispenser that only makes xenobot/arch chems.

**Comment by DrSnips** [comment]
With antitoxin also being used to offset overdose damage of hyperzine it's clear why it's a popular chem not for just chloral resistance. I would like to blame this on xenoarch suits for being super shit and having a huge amount of slowdown. This leads to making a whole lot of hyperzine and to aid it a whole lot of antitoxin. So let's say we cut those two chems out and where does this bring the actual xenoarch chem? Right up to spot number 3.

**Comment by Allspice** [comment]
> The obvious solution is to add a subtype of dispenser that only makes xenobot/arch chems.
Or to replace it with (nothing|a beaker or two of xenoarch chems) like Botany has to deal with.
On a side note, I'm interested in how much synaptizine the outpost disp produces.

**Comment by EenKogNeeto** [comment]
If you remove it, Nothing happens because research can make a replacement in the first 4 minutes of the round anyway, All you do is make it more tedious to fix research after someone wastes all the acid and shit. Keep it there so antags can get their shitterjuice and be caught doing it by the AI. Though I do think you could replace it in terms of nonantags simply with a box of the Xenoarch liquid, Its there for the antags. I break in and use it often as an antag.

**Comment by DrSnips** [comment]
Even with those top 5 chems I'm pretty sure chemistry dispensers in medbay look pretty similar with chloral, antitoxin, and hyperzine in the top 5. No matter what chem dispenser you look at chloral is very likely to come out on top because there's a lot of shitty players and chloral is one of if not the best way to get someone.
Speaking of, maybe it's time for chloral to go.

**Comment by EdXbtkOEwGw** [comment]
@DrSnips For Medbay, the top 5 are Antitox, Bicaridine, Dexalinplus, Kelotane, and Inaprovaline (as precursor to other chemicals most likely)

**Comment by Kurfursten** [comment]
Antitoxin overdose often isn't a thing in real life because it's hard to obtain in large quantities. We farm animals for antitoxin where the antiserum develops inside their blood. I _would_ suggest you make antitoxin a relatively precious material by forcing you to draw snakeblood from Corpus or a cargo-ordered snake (add an antiserum refill crate like the blood crate too for the lazy), but half the meds chemistry makes rely on antitoxin as a base material and I don't want to be crucified particularly.

_Instead_, why not just add organ failure to antitoxin as an overdose effect? Easy enough, no complex systems to keep track of, overdoses are not a new mechanic in medbay, and most doctors are already familiar with removing someone's liver because they shotgunned 30 bar drinks.

**Comment by Kammerjunk** [comment]
Overdoses aren't a new mechanic, but Dragonbro reworked them a while ago. Making a reagent do organ damage or more toxin damage than it can heal or pretty much anything you like is pretty simple now

**Comment by Blithering** [comment]
Could just make antitoxin overdoses make you puke, no-one likes being stunned and throwing up their immortalitymix, plus it would still have fringe uses in actual medbay
That said, the outpost chem dispenser is a hotspot for traitor activity partly because it's so isolated, noones gonna be watching what you make down there

**Comment by Aurx** [comment]
Short of putting an ID lock on the chem dispenser (speaking as a chemist, please do this) you're not going to stop any but the laziest scientists.  R&D is easy, and nobody bats an eye if R&D has a chem dispenser in it.  

This is really just a "should this be break-in central" question.

**Comment by PJB3005** [comment]
> It's no secret that scientists are hyperzine junkies. Hyperzine is made in 50.8% of all rounds, and an average of 122u are made in each round that it gets made. Xenoarchaeology somewhat needs their hyperzine, but there's a Xenoarchaeologist in a lot less than 51% of all rounds... where does the rest come from?

If this is an issue we need to combat hyperzine, not the chem dispenser.

Hyperzine is stupidly broken and I'm amazed nobody has tried nerfing it yet.

> Yikes. People figured out anti-toxin makes you immune to chloral, and so 148491u of anti-toxin have been made in total. It's only made 30.7% of the time, but when it gets made, a few recurrent ckeys literally empty the chemdispenser energy pumping their veins with enough antitoxin to last them for the rest of the round. The powergame is literally out of this world, lads.

Pretty simple solution: make anti-toxin OD and when it does it kicks you so hard you'll need adminordrazine or a cloner to save you.

**Comment by PJB3005** [comment]
Removing the outpost dispenser will do nothing except annoy us roundstart xenoarchs. The powergamers will just build a new one somewhere else in an even *less* traceable way. If you want to remove the powergaming, remove the actual powergaming.

**Comment by Kurfursten** [comment]
I don't think anyone would even consider removing the outpost dispenser and not replacing it with plenty of bottled lithium sodium tungstate. I do think there is something to be said for ease of access though. Just putting something slightly further out of reach will have an affect on how many people use it. For instance, when I added hacking to jukeboxes somebody (Cluster?) requested I make the hack unusually difficult so we wouldn't have syndie playlist every shift, and it remains arcane enough to prevent constant FANFICTION THEATER.

Incidentally, I say all of this strictly within the realm of hypothetical rationality. Don't read that first paragraph to imply I necessarily think the outpost dispenser *needs* to go. I do agree that the highest priority needs to be dealing with chemical abuse rather than a mapping issue (although I disagree somewhat with your priorities, I believe antitoxin's lack of an overdose effect is currently the bigger problem than hyperzine).

**Comment by PJB3005** [comment]
I was just going by the order of the main post. I do also agree anti-tox should be fixed first.

**Comment by ghost** [comment]
>make anti-toxin OD and when it does it kicks you so hard you'll need adminordrazine or a cloner to save you

And now every newbie doctor accidentally giving a little too much antitox their patient fucks their whole life up, thanks to disproportionately sadistic grudgecoding. And then people start intentionally using antitox as a legal superkilldrug.

I mean think, what's the problem? People use antitox to become chloral-immune? Then make antitox not remove chloral when taking over 15u, why the fuck do you have to go to the other end of the spectrum and break a load of shit.

**Comment by Aurx** [comment]
What if the anti-tox OD removed anti-tox?

**Comment by ghost** [comment]
That could work too.
Maybe make people vomit. Periodic stuns would make powergamers have a stroke, but wouldn't be bad if you're in Medbay.

**Comment by RealestEstate** [comment]
>With antitoxin also being used to offset overdose damage of hyperzine it's clear why it's a popular chem not for just chloral resistance.

but overdosing in hyperzine will damage your liver, which will eternally cause you toxin damage until liver is fixed, iirc, so overdosing on hyperzine is a really dumb thing to do even if you're offsetting it with antitox. also it's really not that hard to periodically medicate yourself for that extra powergayman  boost, you can use a spectrometer, but there's even more creative ways to keep track of internal reagents (ie, using seemingly useless flavortext chems as an internal metabolism clock, diluting chemicals to hit just under the OD threshold, etc). The problem with the chem dispenser is that if you can make something, you can make thousands of that thing in a matter of seconds or minutes. this is partly why OD exists. I totally agree that antitoxin should have a debilitating, but not lethal drawback to OD. It's not gonna completely remove massive powergaming, but it will make it require some creativity/effort to do. if you make powergaming trivial, then people are more likely to abuse it. take botany for example. you can essentially make a good portion of the chems that chemsitry can, and some deadlier to boot (you can make 100 unit liquid emps, sold plasteel, solid gold, solid uranium, instakill melee weapons, powerless lights that span 20 tiles each, jaunt in a bag, etc etc), but it's locked behind one of the most unrewarding, time-consuming and tedious game mechanics I can think of in the game (probably tied with genetics, but genetics is quicker to complete) . It took me about a few weeks to learn how to make slippery teleporting cyanide pacid cherries, but it took me a year to learn how to make it in reasonable time to have any effect on the round.
tl;dr I think making powergaming less trivial is almost always a good decision. just because someone CAN do something powergey within the scope of their job or access doesn't mean they will every time. just my thought

**Comment by RealestEstate** [comment]
@Kurfursten lithium tunstate in a beaker has the nasty tendency of getting accidentally spilled, and besides, lithium tungstate has to be mixed with carrier fluids for certain parts of the xenoarch process, carrier fluids like carbon,iron,nitrogen, mercury, etc. just a thought

**Comment by Kurfursten** [comment]
Chemistry seems to manage to avoid spilling their plasma most shifts, at least, but if scientists really are extra bad at handling chemicals, just use a non /glass/ reagent container, because those can't splash. (e.g.: can)

**Comment by RealestEstate** [comment]
ok, so like a lithium tungstate soda vendor, it gives you cans of lithium tungstate. also comes in select flavors like cool carbon, mad mercury, gnarly nitrogen, and intimate iron for all your carrier fluid needs

**Comment by FudgePucker** [comment]
>What is the research outpost chemical dispenser really used for. 'Oh obviously chloral, yup it is'

I think the problem isn't the chemical dispenser, its that most of science is filled with the worst players on the station. They got antag and will murderbone using the infinite resources of science, or power game to shut down antags that aren't blessed with the infinite resources of science all the while praying for autotraitor.

@RealestEstate I really like this idea. Also make the contraband be cans of mutagen.

**Comment by DrSnips** [comment]
A limited chem dispender like soda machines would be ok in my opinion with only the chems needed  to make xenoarch chems. Maybe xenobot chems too.

**Comment by iFlashYou** [comment]
How about we dont just remove everything because someone got once or twice antagonized, in a active or passive way, by it?

**Comment by PJB3005** [comment]
@iFlashYou these stats, if true, aren't "got antagonized once or twice". It's just mass tier powergaming.

**Comment by ghost** [comment]
Just nerf chloral, ez. Make auto injectors have a 50% chance to miss

**Comment by Aurx** [comment]
Don't make the medical tool not work, man.  
Make it not accept new reagents.  The issue isn't "auto injectors", it's "putting chloral into autoinjectors".

**Comment by Kammerjunk** [comment]
Autoinjectors taking new reagents is pretty heavy powercreep anyway IIRC
Just make the hypo the only tool capable of functioning like that
